templates/bridge-infra/agglayer-config.toml

```
@@ -1,6 +1,10 @@
 prover-entrypoint = "{{.agglayer_prover_entrypoint}}"
 debug-mode = true
 
+{{if .mock_verifier}}
+mock-verifier = true
+{{end}}
```

I guess we could write this like this?

```suggestion
mock-verifier = {{ .mock_verifier }}
```